robotcontainer.py

```diff
@@ -87,6 +87,12 @@ def __init__(self) -> None:
         NamedCommands.registerCommand(
             "elevatorAlgaeHigh", ElevatorAlgaeHigh(self.elevator)
         )
+        NamedCommands.registerCommand(
+            "elevatorIntake", ElevatorIntakePosition(self.elevator)
+        )
+        NamedCommands.registerCommand("intakeCoral", IntakeCoral(self.intake))
```

Comment:
Ideally we update this so that the IntakeCoral command does both the arm position, intake running, and elevator position. Not an issue to be fixed in this PR though

Reply:
Also we should add a proper isFinished for intake based on current limits and such

Reply:
nah because if the intaking state ends while the robot is still at the station it'll hit the wall on the way back to idle state, best to just let the driver react to when it is safe to return to an idle state